components/ide/jetbrains/image/status/main.go

```diff
@@ -89,10 +90,27 @@ func main() {
 	log.Info(ServiceName + ": " + Version)
 	startTime := time.Now()
 
-	if len(os.Args) < 3 {
-		log.Fatalf("Usage: %s <port> <kind> [<link label>]\n", os.Args[0])
+	var port string
+	var warmup bool
+
+	if len(os.Args) < 2 {
+		log.Fatalf("Usage: %s (warmup|<port>)\n", os.Args[0])
+	}
+
+	if os.Args[1] == "warmup" {
+		if len(os.Args) < 2 {
+			log.Fatalf("Usage: %s <alias>\n", os.Args[0])
```

Comment:
Just noticed here it should be this instead:
```suggestion
			log.Fatalf("Usage: %s warmup <alias>\n", os.Args[0])
```
Because `%s` is `status`.